test/integrationConfiguration.test.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+/* eslint mocha/no-mocha-arrows: "off" */
+
+'use strict';
+
+const assert = require('assert');
+const path = require('path');
+
+const IntegrationConfiguration = require('../lib/integrationConfiguration.js');
+
+const plainTextFile = path.join(process.cwd(), "test", "resources/plain-text-file.txt");
+const yamlFile = path.join(process.cwd(), "test", "resources/integration.yaml");
+const jsonFile = path.join(process.cwd(), "test", "resources/integration.json");
+const privateKeyFile = path.join(process.cwd(), "test", "resources/privatekey.pem");
+
+describe('integrationConfiguration.js tests', () => {
+
+    const expected = {
+        metascopes: ['Meta Scope 1', 'Meta Scope 2', 'Meta Scope 3'],
+        technicalAccount: {
+            id: 'Tech Id',
+            org: 'IMS Org Id',
+            clientId: 'Client Id',
+            clientSecret: 'Client Secret',
+            privateKey: '-----BEGIN PRIVATE KEY-----\n-----END PRIVATE KEY-----\n'
+        }
+    };
+
+    it('Input file does not exist', async () => {
+
+        let actual;
+        const expected = new Error("Missing required files");
+
+        try {
+            await IntegrationConfiguration.getConfiguration("fake-file.file");
+        } catch (e) {
+            console.log(e);
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
+        assert.strictEqual(actual.message, expected.message);
+
+        try {
+            await IntegrationConfiguration.getConfiguration(jsonFile, "fake-file.file");
+        } catch (e) {
```

Comment:
We are looking for an exception to be thrown. It would be better to use assert.throws (https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message). 

Some errors have timestamps. You can control the timestamp value by using sinon fake timers: https://sinonjs.org/releases/v1.17.6/fake-timers/. This makes the content of the expected error fully known.

Reply:
Would you accept `assert.rejects()` instead?